src/story.test.ts

```diff
@@ -67,12 +67,23 @@ const CSF3Story: XStory = {
 };
 
 const CSF3StoryStrict: XStory<ButtonArgs> = {
-  render: (args) => 'Named Story',
+  render: args => 'Named Story',
   name: 'Another name for story',
   decorators: [storyFn => `Wrapped(${storyFn()}`],
   parameters: { a: [1, '2', {}], b: undefined, c: Button },
   loaders: [() => Promise.resolve({ d: '3' })],
   args: { x: '1' },
+  play: ({ step }) => {
+    step('a step', ({ step: substep }) => {
+      substep('a substep', () => {});
```

Comment:
I see no reason why you would want to use the inner `substep` here, rather than just `step`. Just for testing I suppose?

Reply:
Yeah, unless you think the inner play shouldn't receive a step at all?

Reply:
No, the inner play should receive the same context. Not giving it a step function would only be confusing and not prevent anyone from doing nested steps because they'd just pull it from the outer play context. The inner context is just for convenience.